Sources/CreateTenant/Wait.swift

```diff
@@ -20,7 +20,6 @@ extension HttpClient {
                     name: tenantName,
                     tenantKind: tenantKind
             )
-                    .unwrapHttpResponse()
 
             if case let .error(error) = templateResponse.errorOrValue {
```

Comment:
This is a test comment
```suggestion
            if case let .error(error) = templateResponse.errorOrValue {
            
```

Reply:
Great suggestion!